### ISSUE TITLE
Request for PF1 System: Add additional "slots" to automark tokens

### ISSUE THREAD

**Issue by Koalahund** [BadIdeasBureau/memento-mori#3]
So in Pathfinder 1e we have 3 conditions when hitpoints reach certain numbers.

1. HP = 0 for disabled
2. HP < 0 for dying
3. HP <= constitution mod for dead

While it is possible to automark a token we can only choose one of them.
Would be nice if we can set at least 3 different conditions with different formulas.

**Comment by BadIdeasBureau** [comment]
This is something I would like to add when I have a fair chunk of development time to devote to it (since it would need me to learn a few new skills to implement it nicely), but that's not likely to be for a while.

In the meantime, the Combat Utility Belt module has a component called Triggler which can be used to define arbitrary numbers of triggers. https://github.com/death-save/combat-utility-belt/wiki/triggler

**Comment by Koalahund** [comment]
Yeah, totally forgot about Combat Utility Belt when I found out that the "HP <= -constitution for dead" check wasn't possible.
So I thought lets try this mod.
Found out that I sadly can't check for negative constitution here aswell because "-data.abilites.con.total" is not a valid datapath and the field doesn't accept formulas.


Setup:
![afebd237b4faac96bdb7f26fed0e0c43](https://user-images.githubusercontent.com/85458282/151089635-61f3b3e9-3f9f-4305-8a44-7bc3b7bedff6.png)
Would need an extra setting to set the Static Value (B) result to a negative number while checked so a result of 10 would be -10 and the actor gets marked as dead at -10HP instead of at 10HP.
